File: solver.py
import datetime


totalHoursNeeded = 486;

print("OJT Hour Solver");
print("Assuming that you will render 8 hours per day and no absence starting today. ");
hoursRendered = int(input(f"Enter Rendered Hours: "))
remainingHours = totalHoursNeeded - hoursRendered;
print("Remaining Hours: " + str(remainingHours) + " Hours");

endDate = datetime.date.today() + datetime.timedelta(days=round(remainingHours / 8))
print("End Date: " + endDate.strftime("%B %d, %Y"))

File: solver.java
import java.util.Scanner;

class solver {
    public static void main(String[] args) {
        System.out.println("OJT Hour Solver");

        // The total hours needed to reach the target
        try (Scanner input = new Scanner(System.in)) {
            // The total hours needed to reach the target
            int totalHoursNeeded = 486;
            
            System.out.print("Enter Rendered Hours: ");
            int renderedHours = input.nextInt();
            
            // Calculate the remaining hours
            int remainingHours = totalHoursNeeded - renderedHours;
            System.out.println("Remaining Hours: " + remainingHours);
        }
    }
}

